easybuild/easyconfigs/l/LLVMtc/LLVMtc-20.1.5.eb

```
@@ -0,0 +1,22 @@
+easyblock = 'Bundle'
```

```suggestion
easyblock = 'Toolchain'
```
Practically, I'm not sure how much difference there is but better safe than sorry